app/views/layouts/application.html.erb

```diff
@@ -9,13 +9,18 @@
   </head>
 
   <body>
-    <%= render partial: 'shared/nav' %>
-    <div style='text-align:center'>
+    <% if current_user %>
+      <%= render partial: "shared/user_nav" %>
+    <% else %>
+      <%= render partial: "shared/guest_nav" %>
+    <% end %>
```

Comment:
@s-espinosa @AliSchlereth if we wanted to refactor this if/else logic into the decorator, how would you do that? Is that a thing?